app/src/main/java/com/travelguide/activities/TravelGuideActivity.java

```diff
@@ -104,13 +108,27 @@ public void onCreate(Bundle savedInstanceState) {
         city = "Any";
         group = "Any";
         season = "Any";
+
+        MenuItem item = nvDrawer.getMenu().findItem(R.id.login_fragment);
+        if (Preferences.readBoolean(this, Preferences.User.LOG_IN_STATUS))
```

Comment:
I have encapsulated this inside one method to keep sanity, because we have three places with this exactly same code.